tool/brick_generator/main.dart

```diff
@@ -34,7 +34,10 @@ void main() async {
       var file = _;
 
       try {
-        if (p.extension(file.path) == '.dart') {
+        // Generated file, to dot create it with copyright header
```

Comment:
Is this a typo? Not sure what to dot means haha

Reply:
@renancaraujo
[oh my dislexy](https://github.com/VeryGoodOpenSource/very_good_dart_cli/pull/3/commits/4107e3c6cd7308eb3a8311de1e0ff991915395a8)